package.json

```diff
@@ -1,28 +1,42 @@
 {
   "scripts": {
-    "start": "yarn run build && yarn run serve",
-    "lint-scss": "stylelint _sass/**/*.scss",
-    "lint-nginx": "gixy nginx.conf",
-    "build": "yarn run build-css && yarn run build-js && yarn run build-jekyll",
-    "build-jekyll": "bundle exec jekyll build",
-    "build-css": "node-sass --include-path node_modules _sass --source-map true --output-style compressed --output css && postcss --use autoprefixer --replace css/**/*.css --no-map",
-    "build-js": "mkdir -p js/build/global-nav && cp node_modules/@canonical/global-nav/dist/global-nav.js js/build/global-nav",
-    "watch": "watch -p '_sass/**/*.scss' -c 'yarn run build'",
-    "serve": "bundle exec jekyll serve -P ${PORT} -H 0.0.0.0",
-    "test": "yarn run lint-scss",
-    "clean": "rm -rf node_modules yarn-error.log css js *.log *.sqlite _site/ build/ .jekyll-metadata .bundle"
+    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
```

Comment:
I think `*.sqlite`,  `_site/ build/` & `.jekyll-metadata` are jekyll specific. But as I'm not 100% sure you can keep them as we have them pretty much on every website.

Reply:
No, we have been removing Jekyll and replacing by Eleventy. So it's fine to remove I guess.